perun-openapi/openapi.yml

```diff
@@ -6832,11 +6868,104 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO deleteVo
+  /urlinjsonout/vosManager/deleteVo/pure:
+    post:
+      tags:
+        - VosManager
+      operationId: deleteVo
+      summary: Deletes a VO. If VO contain members, group or resources, it's not deleted and exception is thrown.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO createVo
+  /urlinjsonout/vosManager/deleteVo/withAll:
+    post:
+      tags:
+        - VosManager
+      operationId: deleteVoForce
+      summary: Deletes a VO. If <code>force == true</code> then VO is deleted including members, groups and resources.
+               Otherwise only empty VO is deleted or exception is thrown.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+        - { name: force, description: "Force must be true", schema: { type: boolean },  in: query, required: true }
```

Comment:
There is no need to have two separate methods deleteVo and deleteVoForce, the "force" parameter can be declared "required: false" which makes it optional. Keep just one deleteVo method with the optional parameter.